app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,31 @@
+class PostsController < ApplicationController
+  def index
```

Comment:
You must add a `before_action` statement to ensure that only signed-in user can access to that controller